index.js

```diff
@@ -40,16 +42,24 @@ function Arpping({ timeout = 5, includeEndpoints = false, useCache = true, cache
 }
 
 /**
-* Build array of full ip range (xxx.xxx.x.1-255) given example ip address
+* Build array of full ip range (xxx.xxx.xxx.1-255) given example ip address
 * @param {String} ip
+* @param {Number} mask
 */
-Arpping.prototype._getFullRange = function(ip) {
+Arpping.prototype._getFullRange = function(ip, mask = 24) {
     // don't use default assignment so false-y values are overwritten
     ip = ip || this.myIP;
-    var ipStart = ip.substr(0, ip.lastIndexOf('.') + 1);
+    // add mask to ip eg xxx.xxx.xxx.xxx/yy
+    ip = ip + '/' + mask;
+
+    var range = [];
+    var block = new Netmask(ips);
```

Comment:
@dkranke This should be ip instead of ips.

Reply:
Will fix both issues asap.